shared/lib/files.js

```diff
@@ -48,9 +48,7 @@ const getDirectories = (directoryPath) => {
  */
 const filterPath = (patterns) => (entryPath) => {
   return patterns.every(
-    (pattern) => minimatch(entryPath, pattern, {
-      matchBase: true
-    })
+    (pattern) => minimatch(entryPath, pattern)
```

Comment:
By removing `matchBase: true` we must opt in to child matches using `**/`

This helps catch issues in future directory change PRs